README.md

```diff
@@ -119,6 +119,26 @@ The application also supports a quiet mode, which can be enabled by setting the
   -y, --yaml=<yamlFile>                  The path to the yaml file. (Not needed for the get commands)
 ```
 
+##### [HINT - DEFAULT VALUES FOR SOME PARAMS]
+
+Since the parameters `api`, `organization` and `space` are rarely changed, there is a property file 
+in the directory that defines default values \
+(see `cloud.foundry.cli/src/main/resources/cf_control.properties`).
+
+So the default-values are:
+```
+  -a=api.run.pivotal.io
+  -o=cloud.foundry.cli
+  -s=development
+```
+
+This reduces the number of program parameters to be called.
```

Comment:
Can this property file be located e.g., in the users' home directories? We could copy a template even when using the app for the first time.

I expected to see some `cf-control.jar login` kind of tool. Not a huge fan of hardcoding the values that only work for us in the source code.

Reply:
Good points. 

I aligned with the PO and its fine for now. 

Its also hard to know, how frequently the value will change for the end-user and maybe we should wait until we receive feedback. 
Also, we could align for another solution, I see only the way: 
- set environment variable for the property-file (which make the project more complex, because you have to mantain multiple configurations, and if you put the property to another directory, you will have a bug and need time to find it etc.)
- pass the path of the property-file as parameter, this would reduce the amount of parameters, but it will also lead to problems, either they will always configure the file with the necessary information to execute the command, or they really will just pass the path of the file, because they really only need the default values.

But in the end, the user can always pass -s, -a, -o, or just one of them, or two. 
Or we could also remove this feature and the end user has always to go some "extra meters", which is also fine.

Reply:
> set environment variable for the property-file (which make the project more complex, because you have to mantain multiple configurations, and if you put the property to another directory, you will have a bug and need time to find it etc.)

Well, nothing in the world's free, ever. That little bit of added complexity really improves the usability, so it's probably worth it. You could argue the same way against *every* feature.

> pass the path of the property-file as parameter, this would reduce the amount of parameters, but it will also lead to problems, either they will always configure the file with the necessary information to execute the command, or they really will just pass the path of the file, because they really only need the default values.

As said, I wouldn't pass the path of a properties file *at all*, I'd just check 2-3 additional locations (current directory and somewhere in the users' home directory).

Let's save this for a future user story, shall we?

Reply:
Yes, that sounds good :)

Reply:
I agree with you